Readme

JohnGrey17 opened the pull request.

JohnGrey17 committed:
Added readme

JohnGrey17 committed:
fixed

yevheniisynytsia commented on README.md:
```diff
@@ -0,0 +1,190 @@
+- **GET /api/orders/{orderId}/items**: Get all order items by order ID
+- **GET /api/orders/{orderId}/items/{itemId}**: Get detailed information about a specific order item
+- **PATCH /api/orders/{id}** : Change the status of an order** (admin only)
+Body
```
I would also like to see a part about challenges that you've faced during development and how you overcame those. Also you may add some possible improvements part

yevheniisynytsia commented on README.md:
```diff
@@ -0,0 +1,190 @@
+- **GET /api/orders**: Get user's order history
+- **GET /api/orders/{orderId}/items**: Get all order items by order ID
+- **GET /api/orders/{orderId}/items/{itemId}**: Get detailed information about a specific order item
+- **PATCH /api/orders/{id}** : Change the status of an order** (admin only)
```
let's also include a "real" env file with some mock data so that it is easily created.
You can use you env file but with removed password

yevheniisynytsia commented on README.md:
````diff
@@ -0,0 +1,190 @@
+```
+- **GET /api/orders**: Get user's order history
+- **GET /api/orders/{orderId}/items**: Get all order items by order ID
+- **GET /api/orders/{orderId}/items/{itemId}**: Get detailed information about a specific order item
````
please add instruction how to run your project

JohnGrey17 committed:
Added
-content of presentation with links
-how to run the Application
-postman collection
-challenges faced during development

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 commented:
Have a question can we put gif on the top of Readme ?
and could it be Mr.Bean))

andrii-hoienko commented on README.md:
```diff
@@ -0,0 +1,362 @@
+<a href="#content-of-presentation"><img src="Logo/back.png" width="20" height="20"></a>
+## Technologies
+
+- Java
+- Spring Boot
+- JWT for authentication
+- MapStruct for DTO mapping
+- Swagger for API documentation
+- Docker for containerization
+- Liquibase for database schema management
+- CustomGlobalExceptionHandler for error handling
```
Add some description to technologies, how you used them? for what purpose?

andrii-hoienko commented on README.md:
```diff
@@ -0,0 +1,362 @@
+- Liquibase for database schema management
+- CustomGlobalExceptionHandler for error handling
+
+<a href="#content-of-presentation"><img src="Logo/back.png" width="20" height="20"></a>
```
I don`t see video presentation as well as link to your linkedIn, you can add this in contribution section

andrii-hoienko reviewed (changes requested):
Ver nice, left some minor comments

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font

JohnGrey17 committed:
Fixed font
